### PR TITLE
Fixes #8222: Rudder 3.2 doesn't build on RHEL3

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8222

### CODE REVIEW

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -66,6 +67,15 @@ ifeq ($(shell ../../build-caching get --force-config ./lmdb-source/ name=lmdb-sr
 	../../build-caching put --force-config ./lmdb-source/ name=lmdb-src version=$(LMDB_RELEASE)
 endif
 
+./pcre-source: /usr/bin/wget
+ifeq ($(shell ../../build-caching get --force-config ./pcre-source/ name=pcre-src version=$(PCRE_RELEASE) >/dev/null 2>&1 || echo KO), KO)
+	# Original URL: http://vorboss.dl.sourceforge.net/project/pcre/pcre/8.38/pcre-8.38.tar.gz
+	#$(WGET) -O $(TMP_DIR)/pcre.tgz http://www.normation.com/tarball/pcre/pcre-$(PCRE_RELEASE).tar.gz
```
Comment (jooooooon) [inline rudder-agent/SOURCES/Makefile]: This shouldn't be commented :)

**Comment by jooooooon** [comment]
Aside from that one comment, this looks good to me!

**Comment by peckpeck** [comment]
PR updated

**Comment by jooooooon** [comment]
OK, this is ready for merge. I'll let you merge it, in case you want to test that merge script of yours again :)

**Pull request merged**